ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -455,6 +455,14 @@ def is_quarantine_enabled(cls):
             os.getenv("DD_TEST_QUARANTINE_ENABLED", default=True)
         )
 
+    @classmethod
+    def should_skip_quarantined_tests(cls):
+        if cls._instance is None:
+            return False
+        return cls._instance._api_settings.quarantine.skip_quarantined_tests and asbool(
+            os.getenv("DD_TEST_QUARANTINE_ENABLED", default=True)
```

Comment:
Same with this variable here. If it's something customers can configure, we should add it to configuration.rst (if it's going to be deprecated, we can add a deprecation note in the .rst file too?)

Reply:
Since quarantine is not publicly released as a feature yet, I think it should not be in the user documentation yet.